src/convertToObject.js

```diff
@@ -6,7 +6,25 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  if (sourceString === '') {
+    return {};
+  }
+
+  const styleArray = sourceString.split(';');
+  const obj = {};
+
+  styleArray.forEach((declaration) => {
+    const parts = declaration.split(':');
+
+    if (parts.length === 2) {
+      const key = parts[0].trim();
+      const value = parts[1].trim();
+
+      obj[key] = value;
```

Comment:
The check `parts.length === 2` will not handle cases where there is no key or no value, or where the declaration is not valid CSS. It might be beneficial to add additional validation to ensure that both key and value are present and properly formatted before adding them to the object.